src/common/ceph_strings.cc

```diff
@@ -415,3 +415,6 @@ const char *ceph_osd_backoff_op_name(int op)
 	}
 	return "???";
 }
+
+
+
```

Comment:
can you please remove these lines?

Reply:
ahhh, I missed those. Done!

Reply:
@jecluis, regarding to your question on Teuthology tests, yes as stated here: https://github.com/ceph/ceph/pull/23550/#discussion_r234785540 

The same is also true for `vstart.sh`